src/segment-layers/run_suma_fs_to_surface.sh

```diff
@@ -24,8 +24,8 @@ rm -rf $fs_surf_path/SUMA
 
     # -ldpref check if it take the suma output somewhere else
 
-rm -rf  $suma_output_dir
+\rm -rf  $suma_output_dir
 
-mkdir -p $suma_output_dir
+\mkdir -p $suma_output_dir
 
-mv $fs_surf_path/SUMA $suma_output_dir
+\mv $fs_surf_path/SUMA $suma_output_dir
```

Comment:
is the `\` at the beginning of the line a typo or some bah dark magic that I do not know?

Reply:
The `\` at the start of the command means that any aliasing will be avoided. That makes these commands more stable across systems and time.

--pt

Reply:
Alright, thanks for the explanation